### ISSUE TITLE
repeatforever with sequence doesn't repeat

### ISSUE THREAD

**Issue by bsy6766** [cocos2d/cocos2d-x#17582]
- cocos2d-x version: 3.14.1
- devices test on: Windows 10
- developing environments
   - NDK version: 
   - Xcode version:
   - VS version: 2015 community
   - browser type and version:

Steps to Reproduce:
1. Create new project
2. Create sprite and sequence, then run action ([reference](http://www.cocos2d-x.org/wiki/Actions#An-example-sequence))
```cpp
	// create a sprite
	auto mySprite = Sprite::create("HelloWorld.png");
	mySprite->setPosition(visibleSize * 0.5f);

	// create a MoveBy Action to where we want the sprite to drop from.
	auto move = MoveBy::create(1, Vec2(100, 100));
	auto moveBack = MoveBy::create(1, Vec2(-100, -100));

	// create a delay that is run in between sequence events
	auto delay = DelayTime::create(0.25f);

	// create the sequence of actions, in the order we want to run them
	auto seq1 = Sequence::create(move, delay, moveBack, delay->clone(), nullptr);

	// run the sequence and repeat forever.
	mySprite->runAction(RepeatForever::create(seq1));

	this->addChild(mySprite);
```
I tried with several other ways with RepeatForever and Sequence, it seems to happen when there is more than 1 actions in the sequence.

Creating sequence with only 1 action like

```cpp
	auto seq2 = Sequence::create(rotateBy, nullptr);
```

worked as expected but creating more than 1 like
```cpp
	auto seq3 = Sequence::create(rotateBy, delay, nullptr);
```
caused the same issue.

**Comment by Bilalmirza** [comment]
the above code works fine on iOS and Mac... didn't test it on windows yet.

**Comment by j-jorge** [comment]
This issue is fixed by #17557.
